### ISSUE TITLE
Can't select all roles for the "role reward" section on the server settings (maybe a search bar for all roles could be a way to fix this)

### ISSUE THREAD

**Issue by Priazol** [top-gg/issues#850]
#### Expected Behavior

I expected to find my bot MySuperBot

#### Observed Behavior

Instead I found this cat

#### Applicable Links or Attachments

![Image of the cat](https://top.gg/images/error.jpg)
[Link to MySuperBot](https://top.gg/bots/mysuperbot)

#### Steps to Reproduce

1. Open top.gg
2. ...

#### Web Browser and Version

Netscape navigator 9.0.0.6

#### Operating System and Version

Windows 98 SE

#### Screen Resolution

320×240

#### Additional Information/Comments

It might be because I visited a non existing page

**Comment by Priazol** [comment]
I didn't know if I should delete the text above, never used GitHub before^^

**Comment by bchurton** [comment]
Yep - delete the text above and replace it with the information applicable to your issue
